client/src/actions/dataActions.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+import type {Colab} from '../state'
```

Comment:
I know it's in the api endpoint name, but let's keep our code grammatically correct :slightly_smiling_face: It's collab, not colab.

Reply:
I used "colab" with one L because Google is using the terms "Google Colaboratory" and "Colab" everywhere. I really don't know why but let's see the URL or the title of page or just try to search in Google: [https://colab.research.google.com/notebooks/welcome.ipynb](https://colab.research.google.com/notebooks/welcome.ipynb)

Reply:
Ah, so maybe it's a short from laboratory... Ok then, let's keep it consistent with api.